csharp/device/Microsoft.Azure.Devices.Client/Common/Security/SharedAccessSignatureBuilder.cs

```diff
@@ -15,6 +18,10 @@ namespace Microsoft.Azure.Devices.Client
 #else
     using System.Security.Cryptography;
 #endif
+
+#if !NETMF
```

Comment:
See the next comment. If you remove this and at latter point someone decides to use the crypto API from NETMF they'll have to add the using. Until then it's not necessary so it's OK to remove that.